cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/CruiseControlMetricsProcessorTest.java

```diff
@@ -57,62 +57,79 @@ public void testBasic() {
     metrics.forEach(processor::addMetric);
 
     MetricSampler.Samples samples =
-        processor.process(getCluster(), Arrays.asList(T1P0, T1P1, T2P0, T2P1), MetricSampler.SamplingMode.ALL, METRIC_DEF);
+        processor.process(getCluster(), Arrays.asList(T1P0, T1P1, T2P0, T2P1), MetricSampler.SamplingMode.ALL);
 
     assertEquals(4, samples.partitionMetricSamples().size());
     assertEquals(2, samples.brokerMetricSamples().size());
 
     for (PartitionMetricSample sample : samples.partitionMetricSamples()) {
       if (sample.entity().tp().equals(T1P0)) {
-        validatePartitionMetricSample(sample, _time.milliseconds() + 2, 1.27610208, 20.0, 100.0, 100.0);
+        validatePartitionMetricSample(sample, _time.milliseconds() + 2, 1.27610208, 20.0, 80.0, 100.0);
       } else if (sample.entity().tp().equals(T1P1)) {
-        validatePartitionMetricSample(sample, _time.milliseconds() + 2, 18.5758513, 500.0, 1000.0, 300.0);
+        validatePartitionMetricSample(sample, _time.milliseconds() + 2, 18.5758513, 500.0, 500.0, 300.0);
       } else if (sample.entity().tp().equals(T2P0)) {
-        validatePartitionMetricSample(sample, _time.milliseconds() + 2, 20.0116009, 400.0, 1050.0, 200.0);
+        validatePartitionMetricSample(sample, _time.milliseconds() + 2, 20.0116009, 400.0, 650.0, 200.0);
       } else if (sample.entity().tp().equals(T2P1)) {
-        validatePartitionMetricSample(sample, _time.milliseconds() + 2, 20.0116009, 400.0, 1050.0, 500.0);
```

Comment:
Just out of curiosity, why did we change these values?

Reply:
This is because the interpretation of the input values are now different. i.e. previously bytes out means consumer_bytes_out + follower_bytes_out. Now it only means consumer_bytes_out.